### ISSUE TITLE
DOC: Add user guide sections

### ISSUE THREAD

**Issue by inoelloc** [DassHydro/smash#184]
Currently, the user guide is very short and only gives you a quick introduction to smash. Other sections need to be added to the user guide. There is already an issue about bayesian estimation case study #128

**Comment by pag13** [comment]
I agree, before reaching a didactical and more complete user guide, few ideass of tutorials with existing features:

- tuning of the complexity of the regionalization ANN ; others
- multi gauge calibration
- use of camels or other data bases
- simulation with continuous evapo ?
- ...

**Comment by pag13** [comment]
Also low complexity tutorials for beginners (not a hydrology course but simple practice associated to..)
